static/js/activity.js

```diff
@@ -263,6 +278,11 @@ exports.update_users = function (user_list) {
         $('#user_presences').html(templates.render('user_presence_rows', {users: user_info}));
     }
 
+    // If some users are starred then put a <hr> between starred and unstarred users.
+    if ( starred_users.length !== 0 && unstarred_users.length !== 0) {
+        $('li.user_sidebar_entry:nth-child('+starred_users.length+')').after('<hr>');
```

Comment:
I think this `<hr>` should be a part of the template, instead of being added later through jQuery